lib/routes/50forum/zhuanjia.js

```diff
@@ -0,0 +1,47 @@
+const got = require('@/utils/got');
+const cheerio = require('cheerio');
+
+module.exports = async (ctx) => {
+    const response = await got({
+        method: 'get',
+        url: `http://www.50forum.org.cn/home/article/index/category/zhuanjia.html`,
+    });
+    const data = response.data;
+    if (!data) {
+        return;
+    }
+    const $ = cheerio.load(data);
+    const article_list = $('div.container div.list_list.mtop10 ul li a').toArray();
+
+    const out = await Promise.all(
+        article_list.map(async (article) => {
+            const $item = $(article);
+            const link = 'http://www.50forum.org.cn' + $item.attr('href');
+            const reg = /^(.+)\[(.+)\](.+)$/;
+            const keyword = reg.exec($item.text().trim());
+
+            const description = await ctx.cache.tryGet(link, async () => {
+                const result = await got.get(link);
+
+                const $ = cheerio.load(result.data);
+
+                return $('div.list_content').html();
+            });
+
+            const item = {
+                description,
+                title: keyword[1],
+                author: keyword[2],
+                link,
+                pubDate: keyword[3],
+            };
+            return Promise.resolve(item);
```

Comment:
```suggestion
            return item;
```
`return item` is fine.
`Promise.resolve` is not needed.
Or simply return `{
title: ...,
..., etc
}`